examples/structured_2d_dgsem/elixir_advection_smview.jl

```diff
@@ -0,0 +1,122 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Coupled semidiscretization of two linear advection systems using converter functions
+# and mesh views for the semidiscretizations. First we define a parent mesh
+# for the entire physical domain, then we define the two mesh views on this parent.
```

Comment:
OK. I think it might be useful to add the same elixir with two meshes then to allow for a comparison of the results, but I'll leave it up to you to decide whether it might be useful or not

Reply:
Not sure how useful another elixir would be. We have `elixir_advection_coupled.jl` and `elixir_advection_basic.jl` for comparison. Unless we aim for binary comparisons I think it is better not to overpopulate the examples directory.

Reply:
OK, fair enough /r